.runger-config.yml

```diff
@@ -4,5 +4,5 @@ spec-commands: |
   export RAILS_ENV=test PALLETS_CONCURRENCY=${PALLETS_CONCURRENCY:-5} DISABLE_SPRING=1
   export POSTGRES_USER=david_runger POSTGRES_HOST=localhost
   set -x
-  redis-cli -n 8 FLUSHDB
+  redis-cli -n 10 FLUSHDB
```

Comment:
Increasing to 10 so as not to conflict with the DBs in `app/poros/redis_options.rb`. We aren't actually using `8` yet, because RunFeatureTestsB is sharing the `_api` DB, but we might in the future.